homeassistant/components/simplisafe/__init__.py

```diff
@@ -66,6 +72,14 @@
 DEFAULT_WATCHDOG_SECONDS = 5 * 60
 
 WEBSOCKET_EVENTS_REQUIRING_SERIAL = [EVENT_LOCK_LOCKED, EVENT_LOCK_UNLOCKED]
+WEBSOCKET_EVENTS_TO_TRIGGER_HASS_EVENT = [
+    EVENT_CAMERA_MOTION_DETECTED,
+    EVENT_DOORBELL_DETECTED,
+    EVENT_ENTRY_DETECTED,
+    EVENT_MOTION_DETECTED,
+    EVENT_SENSOR_NOT_RESPONDING,
```

Comment:
Why these? shouldn't these be reflected in changing the `available` property of sensors ?

Reply:
It’s tricky. We don’t track sensors in general (see my attempt here: https://github.com/home-assistant/home-assistant/pull/30926). We do, indeed, track `unavailable` for locks, but not things like entry or motion sensors. I included these so automations could, say, send a notification when a sensor goes offline.

Reply:
So now with websocket events it is possible to easily track availability though? So you could revive #30926.

Reply:
Great point... I'll need to do some testing to confirm, but that's a lead I'll follow. In the meantime, I'll pull those events out of this PR to unblock it.